### DISCUSSION TITLE
-metrics flag disabled

### DISCUSSION THREAD

**Discussion by rrehlert** [projectdiscovery/nuclei#4334] [Q&A] [answered]
The -metrics flag was disabled in some of the recent updates (nuclei 2.9.15 it was working), not sure if it was intended because i haven't found anything in the changelogs, but when running "nuclei -metrics" it says that the flag is not defined.
![image](https://github.com/projectdiscovery/nuclei/assets/63610519/c5299b9d-e349-4f55-8282-8c2d73cda3bb)

it was also removed from the help, while -mp is still there
![image](https://github.com/projectdiscovery/nuclei/assets/63610519/5a3c8ef0-ca4c-4cea-9591-d8f689a44f72)

**Comment by ehsandeep** [comment, marked as answer]
@rrehlert it is now enabled as default, below line gets printed when running in verbose mode.

```
[VER] Started metrics server at localhost:9092
```

**Comment by rrehlert** [comment]
Oh, I didn't notice it, thank you!

**Comment by rrehlert** [comment]
Sorry to reopen the discussion, but could you check again?
Although the line is printed in the verbose mode, i'm not able to monitor it  via the `watch -t -n5 "curl -s http://localhost:9092/metrics | jq"` command like I used to, tried changing the port but still wasn't able to do it

**Comment by fmunozs** [comment]
Hey, just noticed this and I was wondering, is there any flag to disable metrics? I am trying to reduce the memory requirements and metrics is not something that I'd be interested in having running.
